src/main/java/bio/terra/app/configuration/WebConfig.java

```diff
@@ -25,4 +26,12 @@ public void addResourceHandlers(ResourceHandlerRegistry registry) {
         .addResourceHandler("/webjars/swagger-ui-dist/**")
         .addResourceLocations("classpath:/META-INF/resources/webjars/swagger-ui-dist/4.3.0/");
   }
+
+  @Override
+  public void addCorsMappings(CorsRegistry registry) {
```

Comment:
This may need a special care, like WDS did - create a local Spring profile and set attributes accordingly.

Reply:
I can look at that, that wouldn't be a bad thing, for sure. I wasn't planning to put that much work into optimizing the code of the prototype, but adding more code cleanliness can only be a good thing. I'll look into that.

Reply:
Thinking more about this, this is actually not needed as soon as we move to using the Terra UI (e.g., not in prototype mode), because Terra UI properly handles the CORS stuff. I will add a comment to that effect though.